xsd/netex_part_2/part2_frames/netex_serviceFrame_version.xsd

```diff
@@ -132,6 +132,11 @@ Rail transport, Roads and Road transport
 			<xsd:group ref="RouteInFrameGroup"/>
 			<xsd:group ref="FlexibleRouteInFrameGroup"/>
 			<xsd:group ref="SectionInFrameGroup"/>
+			<xsd:element name="projections" type="projections_RelStructure" minOccurs="0">
```

Comment:
Can we make this uniform with the rest, hence a group?

Reply:
done